landingzones/tests/test_permissions.py

```diff
@@ -27,10 +33,15 @@
 
 # Local constants
 SHEET_PATH = SHEET_DIR + 'i_small.zip'
+TEST_OBJ_NAME = 'test1.txt'
 
 
 class TestLandingZonePermissionsBase(
-    LandingZoneMixin, SampleSheetIOMixin, TestProjectPermissionBase
+    LandingZoneMixin,
```

Comment:
After implementation of `nothing to do` check the whole class was failing in CI. The reason was the lack of files (what caused `nothing to do` case). So I found a possible solution: add files to the landingzones. However it got very messy because of tons of dependencies from taskflow( Maybe there could be another implementation?